# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/DocumentL10n.h
# Commit: 92b87b0f776c
# Full Hash: 92b87b0f776c3049499038d380190351e2371324
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r 878b052ebeaa -r 92b87b0f776c dom/l10n/DocumentL10n.h
--- a/dom/l10n/DocumentL10n.h	Tue Aug 03 16:25:10 2021 +0000
+++ b/dom/l10n/DocumentL10n.h	Tue Aug 03 16:25:10 2021 +0000
@@ -47,12 +47,10 @@
   NS_DECL_ISUPPORTS_INHERITED
   NS_DECL_CYCLE_COLLECTION_CLASS_INHERITED(DocumentL10n, DOMLocalization)
 
-  static RefPtr<DocumentL10n> Create(Document* aDocument, const bool aSync);
+  static RefPtr<DocumentL10n> Create(Document* aDocument, bool aSync);
 
  protected:
-  explicit DocumentL10n(Document* aDocument, const bool aSync);
-  bool Init() override;
-
+  explicit DocumentL10n(Document* aDocument, bool aSync);
   virtual ~DocumentL10n() = default;
 
   RefPtr<Document> mDocument;